x-pack/filebeat/input/netflow/testdata/golden/Netflow-9-Cisco-ASA-2.golden.json

```diff
@@ -169,10 +169,10 @@
           "ingress_interface": 3,
           "initiator_octets": 81,
           "observation_time_milliseconds": "2016-07-21T13:50:33.035Z",
-          "post_nadt_estination_ipv4_address": "192.168.0.17",
-          "post_napdt_estination_transport_port": 80,
-          "post_napst_ource_transport_port": 61776,
-          "post_nast_ource_ipv4_address": "192.168.0.2",
+          "post_napt_destination_transport_port": 80,
+          "post_napt_source_transport_port": 61776,
```

Comment:
`_napt_`?

Reply:
Turns out is a legit name, see [IPFIX field 227 postNAPTSourceTransportPort](https://www.iana.org/assignments/ipfix/ipfix.xml).

Stands for "Network Address Port Translation". Got me scared in there because there's a NATP acronym too 😅